README.md

```diff
@@ -61,15 +54,9 @@ Feel free to check the [issues page](../../issues/).
 ## Show your support
 
 Give a ⭐️ if you like this project!
-
-## Acknowledgments
-
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+    
+![](https://img.shields.io/badge/stars-0.0.1-brightgreen.svg)
 
 ## 📝 License
 
 This project is [MIT](./LICENSE) licensed.
-
```

Comment:
- [ ] Please replace `./LICENSE` by `./MIT.md` to make this this link work because this link is broken :sob:

Reply:
Thank you I have fixed it